Include/internal/pycore_backoff.h

```diff
@@ -95,16 +83,43 @@ advance_backoff_counter(backoff_counter_t counter)
 }
 
 static inline bool
-backoff_counter_is_zero(backoff_counter_t counter)
+backoff_counter_triggers(_Py_BackoffCounter counter)
 {
     return counter.value == 0;
 }
 
-static inline uint16_t
-initial_backoff_counter(void)
+/* Initial JUMP_BACKWARD counter.
```

Comment:
There's a lot of boilerplate for these initial values; I followed your lead for `adaptive_counter_warmup()` and `adaptive_counter_cooldown()`, more or less.